harness/engine_evaluation.py

```diff
@@ -110,19 +110,19 @@ def evaluate_predictions(data: Dict):
             # Attempt to apply prediction
             patch_type = "pred_try"
             if not tcm.apply_patch(task_instance[KEY_PREDICTION], patch_type=patch_type) \
-                and task_instance[KEY_PREDICTION] is not None:
+                and task_instance[KEY_PREDICTION] is not None \
+                and task_instance[KEY_PREDICTION] != "":
```

Comment:
empty patch throws an error, just skip it and log a failure

Reply:
Thanks for the catch! Incorporated this.